ui/app/pages/swaps/awaiting-swap/view-on-ether-scan-link/view-on-ether-scan-link.js

```diff
@@ -18,7 +19,7 @@ export default function ViewOnEtherScanLink({
       onClick={() => global.platform.openTab({ url: blockExplorerUrl })}
     >
       {isCustomBlockExplorerUrl
-        ? t('viewOnCustomBlockExplorer', [blockExplorerUrl])
+        ? t('viewOnCustomBlockExplorer', [blockExplorerBaseUrl])
```

Comment:
Instead of passing in a second "base URL" for custom block explorers, it might be better to update `getBlockExplorerUrlForTx` to accept the custom block explorer URL.

The library used by `getBlockExplorerUrlForTx` was [recently updated](https://github.com/MetaMask/etherscan-link/releases/tag/v2.0.0) with new methods that accept a custom block explorer URL, so this should be easy enough to do. Then we'd get a URL directly to the correct transaction, just as on Etherscan.

Reply:
The purpose of this was to show a shorter url, as the link already correctly goes to the correct transaction. Based on your other comment, maybe this would be best here:
`t('viewOnCustomBlockExplorer', [new URL(blockExplorerUrl).hostname])`

Reply:
Oops, I see, I misread. Yes, perhaps the hostname would be better here.

Reply:
Done in c42554a2d

Reply:
This still appears to be here. Did you forgot to push?

Reply:
oops, pushed now